pepper-apis/dsm-core/src/test/integration/org/broadinstitute/dsm/model/kit/KitFinalScanUseCaseIntegrationTest.java

```diff
@@ -76,4 +78,38 @@ public void processIsBloodKitHasNoTracking() {
         assertEquals(scanError, scanErrors.get(0));
     }
 
+    @Test
+    public void kitFinalScanError() {
+        KitRequestShipping kitRequestShipping = getKitRequestShipping("ddpLabel", "TEST4");
+        kitRequestShipping.setDdpParticipantId("TEST_PARTICIPANT_GUID");
+        kitRequestShipping.setBspCollaboratorParticipantId("TEST_BSP_ID");
+
+        KitTypeDto kitTypeDto = KitTypeDto.builder()
+                .withRequiresInsertInKitTracking(false)
+                .withRequiredRole(0)
+                .withManualSentTrack(true)
+                .withNoReturn(true)
+                .build();
+        int kitTypeId = kitTypeDao.create(kitTypeDto);
+        kitTypeIds.add(kitTypeId);
+
+        kitRequestShipping.setKitTypeId(String.valueOf(kitTypeId));
+        setAndSaveKitRequestId(kitRequestShipping);
+        Integer kitId = kitDao.insertKit(kitRequestShipping);
+
+        kitIds.add(kitId);
+
+        List<ScanPayload> scanPayloads = Arrays.asList(
+                new SentAndFinalScanPayload("ddpLabel", "kitLabelMoreThan14")
+        );
+        DDPInstanceDto ddpInstanceDto = new DDPInstanceDao().getDDPInstanceByInstanceName(TestInstanceCreator.TEST_INSTANCE).orElseThrow();
+        KitPayload kitPayload = new KitPayload(scanPayloads, 94, ddpInstanceDto);
```

Comment:
What's the magic number (94) for?

Reply:
That's a user id who has all the access. Used in most of kit tests

Reply:
Mind making that a constant with a descriptive name?